docs/source/changelog.md

```diff
@@ -8,6 +8,41 @@ command line for details.
 
 ## [Unreleased]
 
+## 0.11
+
+## [0.11.0] - 2020-01-30
+
+The main change in 0.11 is a refactoring of classes to remove mixins,
+reducing the amount of boilerplate needed.
+In addition, there are some fixes to the Azure AD Authenticator.
+
+This release also *removes* the latest Authenticators added in 0.10
+(AzureAdB2COAuthenticator, AWSCognitoAuthenticator),
```

Comment:
I confirmed that YandexPassportOAuthenticator was also part of 0.10.0 and removed.